internal/cmd/run.go

```diff
@@ -87,3 +105,26 @@ func initializeRunOpts(o *globals.GlobalOpts, platform, version string) error {
 	}
 	return nil
 }
+
+// setHomeEnvoyVersion makes sure the $GETENVOY_HOME/version exists and globals.GlobalOpts HomeEnvoyVersion contains it.
+func setHomeEnvoyVersion(o *globals.GlobalOpts) error {
+	// See if there's an existing default version
+	homeVersionFile := filepath.Join(o.HomeDir, "version")
+	if data, err := os.ReadFile(homeVersionFile); err == nil {
+		o.HomeEnvoyVersion = string(data)
```

Comment:
maybe better validate the format of the data here?

Reply:
oh yeah.. true we can't assume people are using the schema. good point.